emcc.py

```diff
@@ -101,8 +101,6 @@
 
 @unique
 class Mode(Enum):
-  PREPROCESS_ONLY = auto()
-  PCH = auto()
   COMPILE_ONLY = auto()
```

Comment:
Maybe it's worth having a comment here that this means only the compiler is invoked, and includes modes that don't actually compile such as preprocessing and PCH

Reply:
Added some comments.